code/modules/mob/mob.dm

```diff
@@ -872,7 +872,10 @@ var/list/slot_equipment_priority = list( \
 
 //this and stop_pulling really ought to be /mob/living procs
 /mob/proc/start_pulling(atom/movable/AM)
-	if( !AM || !src || src==AM || !isturf(AM.loc) )	//if there's no person pulling OR the person is pulling themself OR the object being pulled is inside something: abort!
+	if(src == AM) // Trying to pull yourself is a shortcut to stop pulling
+		stop_pulling()
+		return
+	if( !AM || !src || !isturf(AM.loc) )	//if there's no person pulling OR the object being pulled is inside something: abort!
```

Comment:
Lol! Love the way 3 coders commented on the format of this (which is horrible) and no doubt more read it (including me) before anyone actually noticed it had an impossiblility in the statement :laugh: